parts/k8s/kubernetesmastercustomscript.sh

```diff
@@ -214,6 +217,27 @@ function downloadUrl () {
     echo Executed curl for \"${1}\" $i times
 }
 
+function runAptDaily() {
+    echo `date`,`hostname`, PRE-APT-SYSTEMD-DAILY>>/opt/m
+    /usr/lib/apt/apt.systemd.daily
+    echo `date`,`hostname`, POST-APT-SYSTEMD-DAILY>>/opt/m
+}
+
+retrycmd_get_tarball() {
+    tar_retries=$1; wait_sleep=$2; tarball=$3; url=$4
+    echo "${tar_retries} retries"
+    for i in $(seq 1 $tar_retries); do
+        tar -tzf $tarball
+        [ $? -eq 0  ] && break || \
+        if [ $i -eq $tar_retries ]; then
```

Comment:
nit: if tar_retries == 1, it will do nothing and exit 1.

Reply:
ported from upstream. will just keep it like this.